src/runtime/registry/google-tag-manager.ts

```diff
@@ -13,15 +13,19 @@ declare global {
 }
 export type GoogleTagManagerInput = RegistryScriptInput<typeof GoogleTagManagerOptions>
 
-export function useScriptGoogleTagManager<T extends GoogleTagManagerApi>(_options?: GoogleTagManagerInput) {
+function use(options: GoogleTagManagerInput) {
+  return { dataLayer: window.dataLayers[options.dataLayerName!], google_tag_manager: window.google_tag_manager }
+}
+
+export function useScriptGoogleTagManager(_options?: GoogleTagManagerInput) {
```

Comment:
do we need to bump TPC to fix these type errors or something?

Reply:
yes, i'll put this in draft mode